src/traces/indicator/defaults.js

```diff
@@ -37,17 +37,18 @@ function supplyDefaults(traceIn, traceOut, defaultColor, layout) {
     coerce('vmax', 1.5 * traceOut.value);
 
     // Number attributes
-    var auto = [];
+    var auto = new Array(2);
     var bignumberFontSize;
     if(traceOut._hasNumber) {
         coerce('number.valueformat');
         coerce('number.font.color', layout.font.color);
         coerce('number.font.family', layout.font.family);
-        coerce('number.font.size', 'auto');
-        if(traceOut.number.font.size === 'auto') {
+        coerce('number.font.size');
+        if(!traceOut.number.font.size) {
```

Comment:
That works if `font.size===0` is invalid. Do we accept `font.size===0` elsewhere?

Otherwise `traceOut.number.font.size === undefined` should work too.

Reply:
I went with `traceOut.number.font.size === undefined` in 95de45724b62a7a1c3d22434af838bd8c690cbfe